### ISSUE TITLE
train and test loss  

### ISSUE THREAD

**Issue by soroushr** [jakeret/tf_unet#267]
How is train/test splitting handled and how can it be stored in tf.summary logs (for visualizing in tensorboard)?

Several open issues (see #130, #163, #203) have discussed similar problems but I cannot find any straightforward answer to it. 

The `batch_x` and `batch_y` that are used in verification are apparently already included in the training set, so this seems like an inappropriate test set (@jakeret correct me if I'm wrong). 

Thanks!
